packages/gamut/src/ContentContainer/index.tsx

```diff
@@ -0,0 +1,36 @@
+import cx from 'classnames';
+import React, { ReactHTML } from 'react';
+
+import s from './styles.module.scss';
+
+export type ContentContainerProps = {
+  className?: string;
+  el?: keyof ReactHTML;
+  /** Toggle a containerWide className to be applied with no max-width and smaller padding */
+  wide?: boolean;
+};
+
+export const ContentContainer: React.FC<ContentContainerProps> = ({
+  children,
+  className,
+  el,
+  wide = false,
```

Comment:
Nit: `= false` is a bit unnecessary since it's only ever being checked for truthiness. Also, you can do this instead of `|| 'div'` below.

```suggestion
  el = 'div',
  wide,
```

Reply:
![Truthiness](https://user-images.githubusercontent.com/17210163/78262185-3507a580-74ce-11ea-9c38-d9a085f3ce58.png)